content/en/providers/check-in/sp/_index.md

```diff
@@ -180,103 +163,92 @@ Service Providers is included in the [User Attribute](#user-attributes) section.
 
 ### Attribute-based authorisation
 
-EGI Check-in provides information about the authenticated user that may
-be used by Service Providers in order to control user access to
-resources. This information is provided by the EGI Check-in IdP in the
-[SAML attribute assertion](#attributes). The table below
-lists the SAML attributes that are relevant for user authorisation:
+EGI Check-in provides information about the authenticated user that may be used
+by Service Providers in order to control user access to resources. This
+information is provided by the EGI Check-in IdP in the
+[SAML attribute assertion](#attributes). The table below lists the SAML
+attributes that are relevant for user authorisation:
 
-| Description                                                                                      | SAML Attribute          |
-| ------------------------------------------------------------------------------------------------ | ----------------------- |
-| [VO/group membership/roles of the authenticated user](#vogroup-membership-and-role-information)  | `eduPersonEntitlement`  |
-| [Level of Assurance (LoA)](#level-of-assurance)                                                  | `eduPersonAssurance`    |
+| Description                                                                                     | SAML Attribute         |
+| ----------------------------------------------------------------------------------------------- | ---------------------- |
+| [VO/group membership/roles of the authenticated user](#vogroup-membership-and-role-information) | `eduPersonEntitlement` |
+| [Level of Assurance (LoA)](#level-of-assurance)                                                 | `eduPersonAssurance`   |
 
 ### References
 
-- [Shibboleth SP
-  Configuration](https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPConfiguration)
-- [SimpleSAMLphp Service Provider
-  QuickStart](https://simplesamlphp.org/docs/stable/simplesamlphp-sp)
-- [Simple SAML 2.0 service provider with mod_auth_mellon Apache
-   module](https://github.com/UNINETT/mod_auth_mellon)
+- [Shibboleth SP Configuration](https://wiki.shibboleth.net/confluence/display/SHIB2/NativeSPConfiguration)
+- [SimpleSAMLphp Service Provider QuickStart](https://simplesamlphp.org/docs/stable/simplesamlphp-sp)
+- [Simple SAML 2.0 service provider with mod_auth_mellon Apache module](https://github.com/UNINETT/mod_auth_mellon)
```

Comment:
@gwarf should I disable the line length check for the links too?
for example in these lines

Reply:
so I think/hope that for links it shouldn't be a problem as we are not expected to shorten them/split them over lines, but let me double check.

Reply:
We will see how it works once the tables will have been fixed, I expect it's the main problem.

Reply:
Great, so all seems fine with long URL.